files/jira.rb

```diff
@@ -4,6 +4,24 @@
 
 class Jira < BaseHandler
 
+  def create_issue_loop(client, issue_json)
```

Comment:
It does protect against transient errors or time outs. Spawning a second event if this handler fails sounds good, would it just re-use the same event data and use a different handler?